lib/generators/rest_api_generator/templates/rest_api_controller.rb.tt

```diff
@@ -1,3 +1,6 @@
+<% if versioning? -%>
+<%= version %>
+<% end -%>
 class <%= class_name.pluralize %>Controller < ApplicationController
```

Comment:
Here we need indentation for the class, rails uses a function called `model_namespacing` for this feature, i think we should follow them